app/src/main/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/VoiceAnalysisService.kt

```diff
@@ -34,7 +35,7 @@ class VoiceAnalysisService : IntentService("VoiceAnalysisService") {
         VoiceAnalysisOption.MachineLearning
 
     @Inject
-    internal lateinit var notifyBabyCryingUseCase: NotifyBabyCryingUseCase
+    internal lateinit var notifyBabyEventUseCase: NotifyBabyEventUseCase
```

Comment:
note fore improvement VoiceAnalysisService isn't an IntentService actually , it doesn't make much sense to use it if intents are not used.

Reply:
Yup it was inherited from the old code base